server-common/src/main/java/org/apache/gravitino/server/authorization/jcasbin/JcasbinAuthorizer.java

```diff
@@ -98,6 +98,26 @@ public void handleRolePrivilegeChange(Long roleId) {
     enforcer.deleteRole(String.valueOf(roleId));
   }
 
+  @Override
+  public void handleMetadataOwnerChange(
+      String metalake, Long oldOwnerId, NameIdentifier nameIdentifier, Entity.EntityType type) {
+    MetadataObject metadataObject = NameIdentifierUtil.toMetadataObject(nameIdentifier, type);
+    Long metadataId = null;
+    try {
+      metadataId = MetadataIdConverter.getID(metadataObject, metalake);
+    } catch (IOException e) {
```

Comment:
Could we move the logic into `MetadataIdConverter.getID`?
```
    try {
    } catch (IOException e) {
       throw new RuntimeException(e);
    }
```

Reply:
fixed